src/shared/components/organisms/navbar/Navbar.tsx

```diff
@@ -0,0 +1,53 @@
+import { Button } from 'antd';
+import cn from 'classnames';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { showDrawer } from '../../../../redux-features/common';
+import styles from './navbar.module.scss';
+
+export interface INavbarItem {
+  name: string;
+  url: string;
+}
+
+export interface INavbar {
+  isUserLoggedin: boolean;
+  navbarItemList: Array<INavbarItem>;
+}
+
+const Navbar: React.FC<INavbar> = ({ isUserLoggedin, navbarItemList }) => {
+  const dispatch = useDispatch();
+
+  const handleButtonClick = () => {
+    if (isUserLoggedin) {
+    } else {
+      dispatch(showDrawer('login'));
+    }
+  };
+
+  const getNavbarItemView = (navItem: INavbarItem, index: number) => {
+    return (
+      <li key={index}>
+        <Link to={navItem.url}>{navItem.name}</Link>
+      </li>
+    );
+  };
+
+  return (
+    <div className={cn(styles.navbarWrapper, styles.flexWrapper)}>
+      <div className={styles.logo}></div>
+      <div className={styles.navItems}>
+        <ul className={styles.navList}>{navbarItemList.map(getNavbarItemView)}</ul>
+      </div>
+      <div className={styles.link}>
+        <Button className={styles.btnLink} type="link" onClick={handleButtonClick}>
```

Comment:
have two buttons - one for each user action. Based on user has logged in or not, we can conditional renders.